### PR TITLE
{lib}[*/2015a] Xerces 3.1.2 (REVIEW)

### PR DESCRIPTION
**Pull request by verdurin**
Simple version update for Xerces-C++

### CODE REVIEW

File: easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.1.2-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+sources = ['xerces-c-%(version)s.tar.gz']
+source_urls = [
+    'http://apache.belnet.be/xerces/c/3/sources/'
+]
+
+dependencies = [
+    ('cURL', '7.40.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.1.2-foss-2015a.eb]: bump to 7.44.0?

File: easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.1.2-intel-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+sources = ['xerces-c-%(version)s.tar.gz']
+source_urls = [
+    'http://apache.belnet.be/xerces/c/3/sources/'
+]
+
+dependencies = [
+    ('cURL', '7.40.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.1.2-intel-2015a.eb]: bump to 7.44.0?

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by verdurin** [comment]
Okay, I need to update cURL separately. Will do. Would you prefer that in a separate PR, or added to this one?

**Comment by boegel** [comment]
@verdurin: just include them here; just as long as you stick to a handful of easyconfigs, that's fine

File: easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.1.2-foss-2015a.eb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+easyblock = 'ConfigureMake'
+
+name = 'Xerces-C++'
+version = '3.1.2'
+
+homepage = 'http://xerces.apache.org/xerces-c/'
+description = """Xerces-C++ is a validating XML parser written in a portable subset of C++. Xerces-C++ makes it easy to give your application the ability to read and write XML data. A shared library is provided for parsing, generating, manipulating, and validating XML documents using the DOM, SAX, and SAX2 APIs."""
```
Comment (boegel) [inline easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.1.2-foss-2015a.eb]: @verdurin: please break up this long line?

File: easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.1.2-intel-2015a.eb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+easyblock = 'ConfigureMake'
+
+name = 'Xerces-C++'
+version = '3.1.2'
+
+homepage = 'http://xerces.apache.org/xerces-c/'
+description = """Xerces-C++ is a validating XML parser written in a portable subset of C++. Xerces-C++ makes it easy to give your application the ability to read and write XML data. A shared library is provided for parsing, generating, manipulating, and validating XML documents using the DOM, SAX, and SAX2 APIs."""
```
Comment (boegel) [inline easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.1.2-intel-2015a.eb]: @verdurin: please break up this long line?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7b5f6a43d75c7f6b8ade for a full test report.

**Comment by boegel** [comment]
going in, thanks @verdurin!

**Pull request merged**